src/ubuntu/18.04/android/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM mcr.microsoft.com/powershell:6.2.1-ubuntu-18.04
```

Comment:
Much of this Dockerfile looks to be a copy of the coredeps Dockerfile.  Would it be better to change the base image to `mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-coredeps` and then just install the extra stuff not installed by that base image?